lib/GLPI/Agent/Task/Inventory/Linux/AntiVirus/Bitdefender.pm

```diff
@@ -0,0 +1,81 @@
+package GLPI::Agent::Task::Inventory::Linux::AntiVirus::Bitdefender;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use UNIVERSAL::require;
+use Cpanel::JSON::XS;
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    return canRun('bduitool');
```

Comment:
Are you sure `bduitool` is in the PATH ? If not, you need to set a full path to the expected one.

Reply:
You're right. It is not in the path by default. Right path is 
/opt/bitdefender-security-tools/bin/bduitool

I'll try to edit it

Reply:
This one is also done.